src/repo/zcl_abapgit_dot_abapgit.clas.abap

```diff
@@ -47,22 +47,26 @@ CLASS zcl_abapgit_dot_abapgit DEFINITION
     METHODS get_starting_folder
       RETURNING
         VALUE(rv_path) TYPE string .
-
     METHODS get_folder_logic
       RETURNING
         VALUE(rv_logic) TYPE string .
-
     METHODS set_folder_logic
       IMPORTING
         !iv_logic TYPE string .
-
     METHODS set_starting_folder
       IMPORTING
         !iv_path TYPE string .
-
     METHODS get_master_language
       RETURNING
         VALUE(rv_language) TYPE spras .
+    METHODS get_main_language
+      RETURNING
+        VALUE(rv_language) TYPE spras .
+    METHODS get_translation_languages
```

Comment:
similarly here. Although the name is OK from general perspective but it is not very clear in the existing context. Also in some places it is i18n, in some translation ... I'd suggest to unify.

Reply:
renamed to `get_i18n_languages`